src/Library/demos/Toggle Button/main.blp

```diff
@@ -0,0 +1,89 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Toggle Button";
+  description: _("Represent active-state visually");
+
+  Box {
+    orientation: vertical;
+
+    Box Grouped_Toggle_Buttons {
+      orientation: horizontal;
+      halign: center;
+      margin-bottom: 12;
+      styles ["linked"]
+
+      ToggleButton button_first {
+        active: true;
+        icon-name: "eye-not-looking-symbolic";
+      }
+
+      ToggleButton button_second {
+        active: false;
+        icon-name: "eye-open-negative-filled-symbolic";
+        group: button_first;
+      }
+    }
+
+    Label {
+      label: "Grouped Toggle Buttons";
+      justify: center;
```

Comment:
this does nothing
in general - remove properties which don't do anything (unless we need them for examples)